pages/Javascript-API.md

```diff
@@ -49,27 +49,18 @@ Returns the active camera.
 #### `getFeatureAt(pixel)`
 Simple object-picking may be enabled by setting any layer's `interactive` parameter to `true`. This will enable Tangram's "feature selection" capability for objects in that layer. These objects can then be queried with the `getFeatureAt()` function, which takes pixel coordinates within the map view in the form `{ x, y }`, and returns a promise containing the feature (if any) at those pixel coordinates (if multiple features are drawn at that location, only the top-most one is returned).
 
-The promise resolves with a `selection` object containing `{ feature, changed }` properties. If `feature` is present, `feature.properties` will contain its properties from the original data source, and `feature.layers` will contain a list of layer names which contained the feature; if `feature` is undefined, no feature was found. The `changed` flag indicates if the selected feature changed since the last query.
+The promise resolves with a `selection` object:
 
 ```javascript
-> map.on('click', function() {
->    var pixel = { x: event.clientX, y: event.clientY };
->    layer.scene.getFeatureAt(pixel).then(function(selection) {
->       console.log(selection.feature, selection.changed);
->    });
->});
+{ feature, changed, pixel, leaflet_event }
 ```
 
-```javascript
-> scene.getFeatureAt(pixel).then(function(selection) {
->    if (selection.feature) {
->       console.log(selection.feature.layers);
->    }
-> });
-<- ["roads:major_road:trunk_primary:early", 
-"roads:major_road:trunk_primary:routes:early", 
-"roads:major_road:trunk_primary:labels-trunk_primary-z13"]
-```
+- `feature`: when present, `feature.properties` will contain the feature's properties from the original data source; if `feature` is undefined, no feature was found.
+- `changed`: a flag indicating whether the selected feature changed since the last query
+- `pixel`: the XY location within the map container where the event occurred, in the form `{ x, y }`
+- `leaflet_event`: the Leaflet event that triggered the selection
+
+Feature picking is accomplished by assigning a unique color to each feature onscreen and rendering the scene to an offscreen buffer. When queried, `getFeatureAT()` checks the offscreen render at the given location, and identifies the feature by its color.
```

Comment:
Let's not go into implementation details about feature picking (e.g. no need to talk about the unique color + offscreen render buffer -- I think this is cool but a distraction from someone using this practically).